documentation/faq.mdx

```diff
@@ -69,29 +69,57 @@ No, these games run on completely different engines and were not made by Naughty
 
 No. _Ratchet & Clank_ does not use Naughty Dog's engine and was not written in GOAL, it was developed by a completely different team of people. As stated in staff roll for the _Ratchet & Clank_ games, only the background renderers use Naughty Dog technology. Our work is not transferrable to that series.
 
+### Will you decompile The Last Of Us? Isn't it written in GOAL?
+
+To our konwledge, the four Jak games are the only ones ever written using GOAL. Naughty Dog did demonstrate how they use LISP in The Last Of Us.
+
```

Comment:
Is this really a question people ask? I don't think anybody knows that they use racket/lisp scripts in Last of Us, let alone make the connection with GOAL here (there's an Uncharted-size gap here).

Reply:
I agree. I left it in, because it was already part of the commit, but noted at the pull request that this perhaps should be scraped, since only my own question was related to that.